### PR TITLE
[9.0]statechart: Improve  error message

### PR DESCRIPTION
**Pull request by zakiuu**

### CODE REVIEW

File: statechart/models/statechart_mixin.py [hunk 1]
```diff
@@ -94,7 +94,8 @@ def _sc_execute(self, interpreter, orig_event):
         if not all([step.transitions for step in steps]):
             # at least one step had no transition => error
             raise NoTransitionError(
-                _("Event not allowed.\n\n"
+                _("This action is not allowed in the current state "
+                  "or with your access rights.\n\n"
                   "Original event: %s\nSteps: %s") %
```
Comment (sbidoul) [inline statechart/models/statechart_mixin.py]: s/Original event:/Technical details of the error:/

Can you add i18n too?

File: statechart/i18n/fr_BE.po [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+#, python-format
+msgid "This action is not allowed in the current state "
+"or with your access rights.\n"
+"\n"
+"Technical details of the error: %s\n"
+"Steps: %s"
+msgstr "Cette action n'est pas autorisée dans l'état actuel ou avec vos droits d'accès.\n"
+"\n"
+"Détails techniques de l'erreur: %s\n"
+"Étapes: %s"
```
Comment (sbidoul) [inline statechart/i18n/fr_BE.po]: make this fr.po, not fr_BE.po

**Review by sbidoul — COMMENTED** [review]
Small remark, otherwise looks good

File: statechart/i18n/fr_BE.po (hunk 2 above)
Comment (zakiuu) [reply statechart/i18n/fr_BE.po]: @sbidoul done!

**Pull request merged**